src/components/Action/DemanderUneSubvention.tsx

```diff
@@ -0,0 +1,208 @@
+        closeDrawer={() => {
+          setIsDrawerOpen(false)
+        }}
+        id={drawerId}
+        // Stryker disable next-line BooleanLiteral
+        isFixedWidth={false}
+        isOpen={isDrawerOpen}
+        labelId={labelId}
+        ref={drawer}
+      >
+        <DrawerTitle id={labelId}>
+          <TitleIcon icon="money-euro-circle-line" />
+          <br />
+          Demander une subvention
+        </DrawerTitle>
+        <p className="color-grey">
+          Saisissez le montant de la subvention que vous souhaitez obtenir de l’état.
+        </p>
+        <div className="fr-select-group">
+          <label
+            className="fr-label"
+            htmlFor={selectEnveloppeId}
+          >
+            Enveloppe de financement concernée
+          </label>
+          <select
+            className="fr-select"
+            defaultValue=""
+            id={selectEnveloppeId}
+            onChange={(event) => {
+              setBudgetEnveloppe(enveloppeById[event.target.value].value)
+            }}
+          >
+            <option
+              disabled
+              value=""
+            >
+              Choisir
+            </option>
+            {enveloppes.map((enveloppe) => (
+              <option
+                key={enveloppe.id}
+                value={enveloppe.id}
+              >
+                {enveloppe.label}
+              </option>
+            ))}
+          </select>
+        </div>
+        <label
+          className="fr-select-label"
+          htmlFor={inputMontantPrestaId}
+        >
+          Montant en prestation de service
+        </label>
+        <input
+          className="fr-input"
+          disabled={!isBudgetEnveloppe}
+          id={inputMontantPrestaId}
+          max={montantMaximal(montantRh)}
+          min="0"
+          onInput={(event) => {
+            setMontantPresta(Number(event.currentTarget.value))
+          }}
+          type="number"
+        />
+        <br />
+        <label
+          className="fr-label"
+          htmlFor={inputMontantRhId}
+        >
+          Montant en ressources humaines
+        </label>
+        <p className="fr-text--sm color-grey fr-mt-1w">
+          Il s’agit d’une ressource humaine interne à la structure employeuse faisant partie de la
+          gouvernance et récipiendaire des fonds. Format attendu : Montant ETP en euros
+        </p>
+        <input
+          className="fr-input"
+          disabled={!isBudgetEnveloppe}
+          id={inputMontantRhId}
+          max={montantMaximal(montantPresta)}
+          min="0"
+          onInput={(event) => {
+            setMontantRh(Number(event.currentTarget.value))
+          }}
+          type="number"
+        />
+        <ul
+          className={`background-blue-france color-blue-france fr-mt-4w fr-mb-4w fr-pt-2w fr-pr-2w fr-pb-2w ${styles['no-style-list']}`}
+        >
+          {isBudgetEnveloppe ? (
+            <li className={styles['horizontal-text-input']}>
```

Comment:
```suggestion
            <li className="fr-grid-row space-between">
```
Ca ne fait pas le job ? Plutôt que d'avoir notre propre classe